modules/kubernetes/trivy/templates/trivy-operator-values.yaml.tpl

```diff
@@ -10,7 +10,17 @@ trivy:
   mode: ClientServer
   severity: MEDIUM,HIGH,CRITICAL
   ignoreUnfixed: true
-  serverURL: "http://trivy.trivy.svc.cluster.local:4954"
+  %{~ if provider == "aws" ~}
+  serviceAccount:
+    annotations:
+      eks.amazonaws.com/role-arn: ${trivy_operator_role_arn}
```

Comment:
Hmm isn't there a service account created for the trivy server?
You will probably need to use var.trivy_role_arn this because the role that gets created is specific to the serviceAccount.
You might also need to update the role_arn to match the new serviceAccount that gets created by the oeprator.

Reply:
From what i can tell, trivy-server uses the SA created for trivy-operator.
What do you mean with "update the role_arn to match the new serviceAccount that gets created by the oeprator." Which role is this?

Reply:
Currently there are 2 arn's,`trivy_operator_role_arn`  and `trivy_role_arn`.
There should be no use for both?

Reply:
Ahh okay if there is only one SA, you can remove the one called `trivy_role_arn`  just as you say.
https://github.com/XenitAB/terraform-modules/blob/1aff96b7e23bb6038b89582c84eab990cc21dbcf/modules/aws/eks/iam.tf#L248

Good job